cloud/template.yml

```diff
@@ -31,3 +31,58 @@ Resources:
                   - 'dynamodb:Query'
                 Resource: '*'
   # End IAM Roles
+
+  userTable:
```

Comment:
Just like I've done for the IAM roles, I think we should have a comment saying Start DynamoDB Tables and one for the end. In the future, we might have to create separate yml files but for now, clearly separating them by comments is fine.